app/authentication_views.py

```diff
@@ -1,24 +1,27 @@
-from flask import render_template, request, session, redirect, url_for
+from flask import render_template, request, redirect, url_for
 from app import app
 
 from app.src.utilities import crypto
 from app import register_validation as validation
 from app.src.repositories import UsersRepository
 
 from app.src.utilities.logger import logger
-
+from app.src.utilities import logginManager
```

Comment:
loggin? you mean logging? likelogging in?
word logging alone can be easily misunderstood as log manager
rename to authenticationManager / authManager / sessionManager or something

Reply:
changed to loginManager